tests/testing.h

```diff
@@ -36,7 +36,7 @@
 #include <sstream>
 #include <functional>
 
-#ifdef _WIN32
+#if (defined _WIN32) || (defined SDL2PP_XCODE_GENERATED)
```

Comment:
I'd start with converting some existing test, say test_wav, so differences and possible  improvements are apparent.